forced_photometry/multiband_photometry.ipynb

```diff
@@ -232,7 +151,14 @@
     "    'r_MAGERR_AUTO', 'FLUX_24', 'FLUXERR_24', 'MAG_GALEX_NUV', 'MAGERR_GALEX_NUV',\n",
     "    'MAG_GALEX_FUV', 'MAGERR_GALEX_FUV'\n",
     "]\n",
-    "cosmos_table = Irsa.query_region(coords, catalog=\"cosmos2015\", radius=radius, selcols=','.join(cols))\n",
```

Comment:
Please keep this one even if it's more resourceful atm than the direct TAP query (I'm working on switching out the backend in astroquery). Users are definitely not required to know about TAP, and working with coordinates this way is not really the best practice we try to show anyway (the premise of they don't need to track their coordinates are lost when they are forced to fall back to use the values as is, etc)

Reply:
Thanks @bsipocz,
I did this because the notebook was just failing as it exhausts the available memory on a small compute job.

Reply:
This notebook used to be able to run, are you trying somewhere else, or with a large area?

Reply:
Yes. I am working with @jkrick to run the full area of the catalog.